docs/medical-api/api-reference/fhir/create-patient-consolidated.mdx

```diff
@@ -24,28 +24,67 @@ api: "POST /medical/v1/patient/{id}/consolidated"
   The entry needs to be an array of [FHIR resources](/medical-api/fhir/resources).
 </ParamField>
 
+<Info>
+  At the moment there is a limit to how much data you can send in a single request. You can send up
+  to 50 resources in a single request and the content-length must not exceed 1MB.
+</Info>
```

Comment:
Should we enforce this for Sandbox only? We can figure this out for prod cxs, so less benefit of adding this constraint there

Reply:
Sounds good

Reply:
I'll change this on the downstream PR since this is being worked there (this PR was only updating docs, this behaviour was already in place).

Reply:
https://github.com/metriport/metriport/pull/824/commits/5cb79762841e078b5678ea9b307ede4823a65bf1